### PR TITLE
Key dumping & refactoring

### PR DESCRIPTION
**Pull request by progbeat**

### CODE REVIEW

File: creepy/_detail/private_numbers.py [hunk 1]
```diff
@@ -43,4 +45,5 @@ def get(path, passphrase):
     return private_numbers
 
 
-app.run()
+mlockall(MCL_FUTURE)
+app.run()
```
Comment (romawhite47) [inline creepy/_detail/private_numbers.py:49]: Add newline.

File: creepy/_detail/private_numbers.py [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
     key_bytes = open(path, 'rb').read()
     loaders = [serialization.load_pem_private_key, serialization.load_ssh_private_key]
     backend = backends.default_backend()
-    for i in range(4):
+    if passphrase is None:
+        num_tries = 4
+    else:
+        assert type(passphrase) is SecureString
+        passphrase = bytes(passphrase.__enter__())
+        num_tries = 1 if passphrase is None else 1
```
Comment (romawhite47) [inline creepy/_detail/private_numbers.py:25]: Looks wrong.

Comment (progbeat) [reply creepy/_detail/private_numbers.py:25]: It's correct actually :)

Comment (romawhite47) [reply creepy/_detail/private_numbers.py:25]: num_tries is 1 or 1 ?)

**Review by romawhite47 — APPROVED** [review]
LGTM

**Pull request merged**